deployment/local/unified_api_server.py

```
@@ -0,0 +1,453 @@
+
+    # Save uploaded file temporarily
+    with tempfile.NamedTemporaryFile(delete=False, suffix='.wav') as temp_file:
+        audio_file.save(temp_file.name)
+        temp_path = temp_file.name
+
+    try:
+        # Transcribe audio
+        result = voice_transcriber.transcribe_file(temp_path)
+
+        if not result or not hasattr(result, 'text'):
+            raise RuntimeError("Transcription failed - no text returned")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)